co-log/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 `co-log` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+## 0.3.0 — Jan 14, 2019
```

Comment:
Let's put `unreleased` tag here to specify more explicitly that this version of the library is not released. Usually we tell the date only after we upload library to Hackage.

```suggestion
## Unreleased: 0.3.0
```